cfme/cloud/instance/__init__.py

```diff
@@ -318,7 +318,7 @@ def power_control_from_cfme(self, *args, **kwargs):
                 raise ItemNotFound(
                     'Failed to find instance in table: {}'.format(self.name)
                 )
-            row.check()
+            row.ensure_checked()
```

Comment:
I have only been able to test this by changing the surrounding code so line 321 would be executed. IT tested fine, but there is not a specific test in PRT that covers this update.